src/main/java/uk/gov/laa/ccms/caab/service/NotificationService.java

```diff
@@ -16,13 +25,18 @@
 @RequiredArgsConstructor
 @Slf4j
 public class NotificationService {
+
   private final SoaApiClient soaApiClient;
+  private final S3Template s3Template;
+
+  @Value("${laa.ccms.s3.buckets.document-bucket}")
+  String bucketName;
```

Comment:
Make this final so that it is included in the required args constructor?

Reply:
Done (and moved to S3ApiClient)